src/main/java/edu/kis/powp/jobs2d/command/CommandStatsUpdater.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.logging.Logger;
+
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.command.visitor.CommandTypeCounterVisitor;
+import edu.kis.powp.observer.Subscriber;
+
+public class CommandStatsUpdater implements Subscriber {
+  private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+  private DriverCommandManager driverCommandManager;
+
+  private long operateToCounterSum = 0;
+  private long setPositionCounterSum = 0;
+  private long commandsLoaded = 0;
```

Comment:
The stats were also meant to provide info on the current command. You have only accumulation.